doc/faq.rst

```diff
@@ -81,3 +81,69 @@ HV_RAM_SIZE is changed to 240M
     hex "Size of the RAM region used by the hypervisor"
     default 0x07800000 if PLATFORM_SBL
     default 0x0f000000 if PLATFORM_UEFI
+
+How to change the assigned output for one UOS instead of the default setting ?
+******************************************************************************
+
+BXT HW has three pipes and each pipe can have three or four planes which helps to
```

Comment:
We never talk about `BXT` in our documentation, can we refer to `Apollo Lake` (or `APL`) instead?

It would also be great to have a document that describes the number of pipes/planes available. This is implicitly described in https://projectacrn.github.io/latest/developer-guides/GVT-g-kernel-options.html#i915-domain-plane-owners for Apollo Lake but I know this is not true for KBL, which is the other HW platform we support. Could we add such info (not necessarily in this PR).